ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
Our cmake file states

 cmake_minimum_required (VERSION 2.8.3)

If we are now requiring 2.8.11, we need to change that. We may also need to update some instructions on wiki pages regarding when/whether the default cmake for a distro is modern enough.